betty/project/extension/demo/content_provider.py

```diff
@@ -14,12 +14,12 @@
     label=Plain("Front page content (demo)"),
 )
 class _FrontPageContent(Jinja2TemplateContentProvider):
```

Comment:
Remove the template and return it inline. 

Or instead, consider removing StaticTranslationsConfigurationAttr and allowing any Localizables to be set on configuration. We can then remove these plugins entirely. (do this in a separate PR)